src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2913,4 +2913,10 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="Alternate_OOBE_Run_Title.Text" xml:space="preserve">
     <value>PowerToys Run</value>
   </data>
-</root>
+  <data name="GeneralPage_EnableExperimentation.Description" xml:space="preserve">
+    <value>Only affects Windows Insider builds</value>
```

Comment:
Does this mean it's only _possible_ on insider builds? If so, it should only be _visible_ on insider builds. But there's no check for that. We shouldn't offer things that are impossible.